modules/product/src/main/java/com/opengamma/strata/product/fxopt/FxSingleBarrierOption.java

```diff
@@ -59,15 +60,15 @@
   /**
    * The barrier description.
    * <p>
-   * The barrier level stored in this field must be represented based on the direction of the currency pair in the 
+   * The barrier level stored in this field must be represented based on the direction of the currency pair in the
    * underlying FX transaction.
    * <p>
    * For example, if the underlying option is an option on EUR/GBP, the barrier should be a certain level of EUR/GBP rate.
    */
   @PropertyDefinition(validate = "notNull")
   private final Barrier barrier;
   /**
-   * For a 'out' option, the amount is paid when the barrier is reached; 
+   * For a 'out' option, the amount is paid when the barrier is reached;
```

Comment:
Your IDE should not be configured to automatically remove whitespace like this. (No need to undo the whitespace parts)

Reply:
I followed the dev guide to configured my IDE and just double-checked I did that correctly earlier, I am not sure what is happening.